src/slick.grid.ts

```diff
@@ -979,7 +979,7 @@ export class SlickGrid<TData = any, C extends Column<TData> = Column<TData>, O e
       if (Draggable) {
         this.slickDraggableInstance = Draggable({
           containerElement: this._container,
-          allowDragFrom: 'div.slick-cell, div.' + this.dragReplaceEl.cssClass,
+          allowDragFrom: 'div.slick-cell, div.slick-cell *, div.' + this.dragReplaceEl.cssClass,
```

Comment:
does the `*` in `div.slick-cell *` mean a slick-cell with any kind of other CSS class? I assume it does.... Also let's convert this line to template literals by using backtick to make it more readable
```ts
allowDragFrom: `div.slick-cell, div.slick-cell *, div.${ this.dragReplaceEl.cssClass }`,
```